embulk-core/src/main/bat/selfrun.bat

```diff
@@ -33,6 +33,50 @@ if "%overwrite_optimize%" == "true" (
     )
 )
 
+for /f "usebackq delims=" %%w in (`java -fullversion 2^>^&1`) do set java_fullversion=%%w

```

Comment:
This is code for Windows. but It execute in `/bin/sh`

Simple reproducde code

```
: <<TEST
for /f "usebackq delims=" %%w in (`java -fullversion 2^>^&1`) do set java_fullversion=%%w
TEST

echo "test"
```

```
sh test.bat
test.bat: line 1: 1: command not found
test
```

Probably, You just quote here document part.

```
: <<'TEST'
for /f "usebackq delims=" %%w in (`java -fullversion 2^>^&1`) do set java_fullversion=%%w
TEST

echo "test"
```

```
sh test.bat
test
```

Reply:
Or, doing the same with #670 may be a solution. Will try this later.